.github/workflows/test.yml

```diff
@@ -29,9 +29,9 @@ jobs:
               - .github/workflows/build.yml
               - db-template/**
```

Comment:
should this be `products/db-template/**`?

Reply:
Yup. Done